src/main/java/mate/academy/service/impl/OrderServiceImpl.java

```diff
@@ -20,8 +22,19 @@ public class OrderServiceImpl implements OrderService {
 
     @Override
     public Order completeOrder(ShoppingCart shoppingCart) {
-        Order toPlace =
-                new Order(shoppingCart.getTickets(), shoppingCart.getUser(), LocalDateTime.now());
+        if (shoppingCart == null) {
+            throw new DataProcessingException(
+                    "Internal error, shopping cart object is null");
+        }
+        List<Ticket> tickets = shoppingCart.getTickets();
+        if (tickets.isEmpty()) {
+            throw new DataProcessingException(
+                    "Internal error, there are no tickets in the shopping cart");
+        }
```

Comment:
Would make more sense to throw a common exception as empty ticket list may need to be handled some other way